start-site/src/main/resources/application.yml

```diff
@@ -1036,6 +1036,16 @@ initializr:
           description: Client that connects to a Spring Cloud Config Server to fetch the application's configuration.
           groupId: org.springframework.cloud
           artifactId: spring-cloud-starter-config
+          links:
+            - rel: reference
+              href: https://docs.spring.io/spring-cloud-config/docs/current/reference/html/#_spring_cloud_config_client
+              description: Config Server Client
+            - rel: reference
+              href: https://docs.spring.io/spring-cloud-config/docs/current/reference/html/#config-data-import
+              description: SpringBoot Config Data Import
+            - rel: reference
+              href: https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#boot-features-external-config-files
+              description: Spring Boot External Application Properties
```

Comment:
Do we really need three links here? Links are rendered in a bullet list now so I'd like us to be conservative. 

Along the same lines of our discussion about the default value, if we believe this is an important part of the getting started experience, I'd rather have a section in the reference guide that covers that, eventually linking (with more text) to other part of the reference guide, including that link in Spring Boot, rather than a flat list like this.